server/routes/support/handlers/offenderDetail.ts

```diff
@@ -18,7 +23,33 @@ export default class OffenderDetailRoutes {
       ? await this.communityService.getStaffDetailByStaffCode(probationPractitioner?.staff.code)
       : undefined
     const hdcStatus = _.head(await this.prisonerService.getHdcStatuses([prisonerDetail], user))
+    const licenceSummary = await this.licenceServer.getLatestLicenceByNomisIdsAndStatus([nomsId], [], user)
 
+    let licencecrn = 'Not found'
+    let crd = 'Not found'
+    let releaseDate = 'Not found'
+    let sentenceStartDate = 'Not found'
+    let sed = 'Not found'
+    let led = 'Not found'
+    let tussd = 'Not found'
+    let licensetused = 'Not found'
+
+    if (licenceSummary != null) {
+      const licence = await this.licenceServer.getLicence(licenceSummary.licenceId.toString(), user)
+
+      licencecrn = licence.crn
+      crd = licence.conditionalReleaseDate
+      releaseDate = licence.actualReleaseDate
```

Comment:
`releaseDate` is not a particularly accurate name for this, as CRD is also a release date. We tend to refer to this date by `ard`

Reply:
actualReleaseDate